pom.xml

```diff
@@ -570,16 +570,6 @@
 		</profile>
 	</profiles>
 
-	<pluginRepositories>
```

Comment:
Can we keep this one? It helps if one has to open cq manually and pruned his ~/.m2 repo. If/whed dash-license-tool is available in maven central that would no longer be needed.

Reply:
Sure, if you still want to do that manually this can be kept.
But I want to remark, in case you just want to get the review summary you can just request a review (although all licenses are already vetted) and download the attached summary files.

Reply:
I was thinking of it as some temp measure till things are proven but seeing your responsiveness we don't need this safeguard :)

Reply:
Glad to hear to that. :)
Nevertheless in the worst case you can still revert this change and #1049.
But the more use cases we have the earlier bugs hopefully become visible.